### ISSUE TITLE
Light Mode

### ISSUE THREAD

**Issue by yungslime** [gcarmix/HexWalk#54]
Hi, is there any chance of adding an optional light theme? This should be considered an a11y feature. 

Thanks!

**Comment by gcarmix** [comment]
Yes, I think it can be a good add-on, thank you for your hint,

C.

**Comment by gcarmix** [comment]
I solved it in v2.1.0, please check it
